core/network/PacketSerialize.hpp

```diff
@@ -45,31 +45,35 @@ void serialize(S &s, ChatMessagePacket &packet) {
   s.value1b(packet.header.type);
   s.value4b(packet.header.size);
   s.value4b(packet.timestamp);
-  s.text1b(packet.message, SERIALIZE_512_BYTES);
   s.value4b(packet.player_id);
+  s.text1b(packet.message, SERIALIZE_32_BYTES);
```

Comment:
pourquoi avoir changé pour 32??

```suggestion
  s.value4b(packet.player_id);
  s.text1b(packet.message, SERIALIZE_512_BYTES);
```